raiden/network/proxies/secret_registry.py

```diff
@@ -71,6 +73,22 @@ def __init__(self, jsonrpc_client, secret_registry_address, contract_manager: Co
     def register_secret(self, secret: Secret):
         self.register_secret_batch([secret])
 
+    def register_secret_light(self, signed_tx: SignedTransaction):
+        try:
+            log.debug("register_secret_light called")
+            transaction_hash = self.proxy.broadcast_signed_transaction(signed_tx)
+            self.client.poll(transaction_hash)
+            receipt_or_none = check_transaction_threw(self.client, transaction_hash)
+            if receipt_or_none:
+                log.critical("register_secret_light failed")
+                raise RaidenRecoverableError("registering secret failed")
+        except HTTPError as e:
+            log.warning("register_secret_light failed: transaction malformed", ex=e)
+            raise RawTransactionFailed("Light Client raw transaction malformed")
```

Comment:
how are we handling this exception? 

node should not crash, it must just log the operation since we can have differents light clients sending malformed tx

Reply:
Currently therey are handled in RestAPI.post_register_secret_light

Reply:
probably you will need to do something like what we have in the deposits, close and now settlement case:

```
        try:
            channel_state = self.raiden_api.get_channel(
                registry_address=registry_address,
                token_address=token_address,
                creator_address=creator_address,
                partner_address=partner_address,
                channel_id_to_check=None
            )

        except ChannelNotFound:
            return api_error(
                errors="Requested channel for token {} between {} and {} not found".format(
                    to_checksum_address(token_address),
                    to_checksum_address(creator_address),
                    to_checksum_address(partner_address)
                ),
                status_code=HTTPStatus.CONFLICT,
            )
        except InvalidAddress as e:
            return api_error(errors=str(e), status_code=HTTPStatus.CONFLICT)

        if total_deposit is not None:
            result = self._deposit_light(registry_address, channel_state, total_deposit, signed_approval_tx,
                                         signed_deposit_tx)

        elif state == CHANNEL_STATE_CLOSED:
            result = self._close_light(registry_address, channel_state, signed_close_tx)
        elif state == CHANNEL_STATE_SETTLING:
            if signed_settle_tx:
                result = self._settle_light(registry_address, channel_state, signed_settle_tx)
            else:
                result = api_error(
                    errors="Signed settle transaction is required for state {}".format(state),
                    status_code=HTTPStatus.BAD_REQUEST,
                )
        else:  # should never happen, channel_state is validated in the schema
            result = api_error(
                errors="Provided invalid channel state {}".format(state),
                status_code=HTTPStatus.BAD_REQUEST,
            )
        return result
```
Something like that where you have a try catch and then we manage the error with the api_error response object.

Reply:
I did something like that in RestAPI.post_register_secret_light

Reply:
@JONAF2103 this lgtm, can we resolve this discussion?